product_docs/docs/jdbc_connector/42.5.4.1/05_using_the_advanced_server_jdbc_connector_with_java_applications/01_loading_the_advanced_server_jdbc_connector.mdx

```diff
@@ -12,12 +12,4 @@ legacyRedirectsGenerated:
 
 <div id="loading_the_advanced_server_jdbc_connector" class="registered_link"></div>
 
-The EDB Postgres Advanced Server JDBC driver is written in Java and is distributed in the form of a compiled Java Archive (JAR) file. Use the `Class.forName()` method to load the driver. The `forName()` method dynamically loads a Java class at runtime. When an application calls the `forName()` method, the Java Virtual Machine (JVM) attempts to find the compiled form (the bytecode) that implements the requested class.
-
-The EDB Postgres Advanced Server JDBC driver is named `com.edb.Driver`:
-
-`Class.forName("com.edb.Driver");`
-
-After loading the bytecode for the driver, the driver registers itself with another JDBC class (named `DriverManager`) that's responsible for managing all the JDBC drivers installed on the current system.
-
-If the JVM can't locate the named driver, it reports a `ClassNotFound` exception (which is intercepted with a `catch` block near the end of the program). The `DriverManager` is designed to handle multiple JDBC driver objects. You can write a Java application that connects to more than one database system by way of JDBC. 
+The EDB Postgres Advanced Server JDBC driver is written in Java and is distributed in the form of a compiled Java Archive (JAR) file. Ensure the JAR file is included in your class path and it will automatically be registered when the Java runtime stats up. It will then be used when a application requests a connection URL beginning with the "jdbc:edb:" schema.
```

Comment:
This is my attempt to remove the future tense (and fix a typo). It would be nice to remove the passive voice also.
```suggestion
The EDB Postgres Advanced Server JDBC driver is written in Java and is distributed in the form of a compiled Java Archive (JAR) file. Ensure the JAR file is included in your class path. The driver is automatically registered when the Java runtime starts up. It is used when a application requests a connection URL beginning with the "jdbc:edb:" schema.
```

Reply:
I've taken another pass at it based on your suggestion.